### PR TITLE
verticalHeight depends on the numberOfMonths prop in DayPicker

### PR DESCRIPTION
**Pull request by b0gok**
Related to https://github.com/airbnb/react-dates/issues/469
It's not the best solution for this case, but I think it's better then nothing. Check this, please.

### CODE REVIEW

File: src/components/DayPicker.jsx [hunk 1]
```diff
@@ -721,9 +721,8 @@ export default class DayPicker extends React.Component {
       phrases,
     } = this.props;
 
-    const numOfWeekHeaders = this.isVertical() ? 1 : numberOfMonths;
     const weekHeaders = [];
-    for (let i = 0; i < numOfWeekHeaders; i += 1) {
+    for (let i = 0; i < numberOfMonths; i += 1) {
```
Comment (majapw) [inline src/components/DayPicker.jsx]: Wait there should def still only be one week header.

File: src/components/DayPicker.jsx [hunk 2]
```diff
@@ -751,9 +750,7 @@ export default class DayPicker extends React.Component {
 
     const horizontalWidth = (calendarMonthWidth * numberOfMonths) + (2 * DAY_PICKER_PADDING);
 
-    // this is a kind of made-up value that generally looks good. we'll
-    // probably want to let the user set this explicitly.
-    const verticalHeight = 1.75 * calendarMonthWidth;
+    const verticalHeight = (calendarMonthWidth * numberOfMonths) + (2 * MONTH_PADDING);
```
Comment (majapw) [inline src/components/DayPicker.jsx]: We should still probably leave a comment as to the effect of this number, but yes, I agree this is better.

**Comment by ljharb** [comment]
@b0gok are you not able to reply to the review and update the PR?

**Comment by b0gok** [comment]
@ljharb I decided that this is an outdated problem, but I did not check. If it is relevant, I can update the PR.

**Comment by ljharb** [comment]
Since the issue is still open, it still seems relevant to me.

**Pull request closed without merging**